source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -29,7 +30,9 @@ The procedure on this page provides instructions for:
    - Accessing the MinIO Console using AD/LDAP credentials.
    - Using the MinIO ``AssumeRoleWithLDAPIdentity`` Security Token Service (STS) API to generate temporary credentials for use by applications.
 
-This procedure is generic for AD/LDAP services. Defer to the documentation for the AD/LDAP provider of your choice for specific instructions or procedures on configuration of user identities.
+This procedure is generic for AD/LDAP services.
+Defer to the documentation for the AD/LDAP provider of your choice for specific instructions or procedures on configuration of user identities.
```

Comment:
```suggestion
See the documentation for the AD/LDAP provider of your choice for specific instructions or procedures on configuration of user identities.
```

`Defer` strikes me as too ... posh?

Reply:
I had that thought as well, although it's used elsewhere too. A second 🤔 confirms it. ✂️